util/generate_haddock3_catalog.py

```diff
@@ -47,30 +48,103 @@
 import logging
 from math import isnan
 from pathlib import Path
+import re
 import sys
 from yaml import dump, load, Loader
 
 from haddock.modules import modules_category
-from haddock import config_expert_levels
+from haddock import config_expert_levels, _hidden_level
```

Comment:
Not sure what `_hidden_level` is

Reply:
Hidden level is the level beyond guru and used for experimental or very obscure parameters